src/screens/TopicMemberScreen/index.js

```diff
@@ -154,6 +154,7 @@ const TopicMemberScreen = () => {
     cancelTokenRef.current = axios.CancelToken.source();
   };
 
+  if (isInitialLoading) return null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a loading state check before rendering the component is a good practice for improving user experience by avoiding flickering or incomplete UI during data fetching. However, make sure that `isInitialLoading` is properly set and updated based on your data fetching status.
```diff
+  if (isInitialLoading) return null;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->